rllib/offline/offline_prelearner.py

```diff
@@ -344,3 +358,111 @@ def convert(sample, space):
             episodes.append(episode)
         # Note, `map_batches` expects a `Dict` as return value.
         return {"episodes": episodes}
+
+    def _map_batch_to_episode(
```

Comment:
rename to: `_map_sample_batch_to_episode`

Also: dumb question, but is it necessary to return this dict here with the "episodes" key? We resolve this key anyways right away after calling this method, correct?

Reply:
I agree, this makes it clear that this is not a batch from `ray.data`.

Reply:
In regard to the return value: the idea was to keep this flexible, meaning that users can use the `OfflinePreLearner` in Pipelines where they need simply episodes. But you are right, it is simpler to override the `OfflinePreLearner` and then use this static method in its `__call__` method.

Another use for it is to pass this static method into a `map_batches` when converting column data or `SampleBatch` data to episodes. I will try out, if this works this simple.

Reply:
Yup, so this works:

```python
"""Put this script into your `ray` root folder."""
import functools
from pathlib import Path

import ray
from ray.rllib.offline.offline_prelearner import OfflinePreLearner, SCHEMA

base_path = Path(__file__).parent / "rllib"
sample_batch_data_path = base_path / "tests/data/cartpole/large.json"

ds = ray.data.read_json(sample_batch_data_path.as_posix())

ds = ds.map_batches(
    functools.partial(OfflinePreLearner._map_batch_to_episode, False) 
)

batch = ds.take_batch(10)

print(batch)
```